src/h5web/visualizations/heatmap/IndexAxis.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Axis } from '@vx/axis';
+import { format } from 'd3-format';
+import styles from './HeatmapVis.module.css';
+import { IndexAxisProps, getOrientedAxisProps } from './utils';
```

Comment:
I put the type declaration and the orientation logic in utils. I am unsure if it should be moved here.

Reply:
Seems to complicate things a bit, at least for `tickLabelProps`.

Reply:
Removed.